src/lib/formatter_json.c

```diff
@@ -28,51 +28,54 @@
 #include "progress.h"
 #include "strings.h"
 
-static int json_format = 0;
```

Comment:
are you sure these are not necessary anymore? I don't even remember why we had them in the first place, but I remember they were necessary 🤣

Reply:
yeah, you are right, and I tested this and looks good... maybe it was needed in earlier versions of the implementation and was never removed. I'm ok with removing this now.